onebusaway-enterprise-webapp/src/main/java/org/onebusaway/enterprise/webapp/actions/api/model/RouteResult.java

```diff
@@ -32,11 +35,28 @@
 
 	private List<RouteDirection> directions;
 
+	private List<StopBean> stops;
+
+	private List<String> stopIdsForRoute = new ArrayList<>();
+
 	public RouteResult(RouteBean route, List<RouteDirection> directions) {
 		this.route = route;
 		this.directions = directions;
 	}
 
+	public RouteResult(RouteBean route, List<RouteDirection> directions, StopsForRouteBean stopsForRouteBean) {
```

Comment:
I added a new signature, rather than changing the old one, because I wasn't sure if the old one was still being used in other locations.

Reply:
That's fine.  Looks good